exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -94,37 +143,225 @@ func (e *exporter) serializeMetrics(md pdata.Metrics) ([]string, int) {
 			metrics := libraryMetric.Metrics()
 			for k := 0; k < metrics.Len(); k++ {
 				metric := metrics.At(k)
-				name, normalizationError := normalizeMetricName(e.cfg.Prefix, metric.Name())
-				if normalizationError != nil {
-					dropped++
-					e.logger.Error(fmt.Sprintf("Failed to normalize metric name: %s", metric.Name()))
-					continue
-				}
 
-				var l []string
+				var metricLines []string
 				switch metric.DataType() {
 				case pdata.MetricDataTypeNone:
 					continue
 				case pdata.MetricDataTypeIntGauge:
-					l = serialization.SerializeIntDataPoints(name, metric.IntGauge().DataPoints(), e.cfg.Tags)
+					for x := 0; x < metric.IntGauge().DataPoints().Len(); x++ {
+						dp := metric.IntGauge().DataPoints().At(x)
+						dims := labelsToDimensions(dp.LabelsMap())
+
+						line, err := serializeMetric(
+							metric.Name(),
+							dtMetric.WithPrefix(e.cfg.Prefix),
+							dtMetric.WithDimensions(
+								dimensions.MergeLists(
+									e.defaultDimensions,
+									dimensions.NewNormalizedDimensionList(dims...),
+									e.staticDimensions,
+								),
+							),
+							dtMetric.WithTimestamp(dp.Timestamp().AsTime()),
```

Comment:
Are metrics ingested correctly with this? I had problems before with time zones and DST to timestamp conversion...

Reply:
`AsTime` doesn't return a UTC timestamp, but an actual `time.Time` object. I'm not sure there's much we can do if the timezone is incorrect